hazelcast/src/main/java/com/hazelcast/multimap/impl/MultiMapService.java

```diff
@@ -338,4 +339,16 @@ public void rollbackTransaction(String transactionId) {
     public void dispatchEvent(EventData event, EntryListener listener) {
         dispatcher.dispatchEvent(event, listener);
     }
+
+    @Override
+    public Map<String, LocalMultiMapStats> getStats() {
+        Map<String, LocalMultiMapStats> mapStats = new HashMap<String, LocalMultiMapStats>();
+        for (int i = 0; i < nodeEngine.getPartitionService().getPartitionCount(); i++) {
```

Comment:
Sorry, I don't know or see a direct way of extracting names. At least you can avoid redundant `createStats(name)` calls by checking if map contains that name or not.

```java
if (!mapStats.containsKey(name)) { 
    mapStats.put(name, createStats(name));
}
```